Ins 27092 versioning docs (#287)
IBM/universal-connectors: docs/Guardium Insights/3.2.x/README.md

# Guardium Insights 3.2.x
The Universal Connector uses plug-in to the datasource server.

There are [available plugins](docs/available_plugins.md) that are ready for connecting the datasource to Guardium Insights.

In case that the requested datasource isn't available, please refer [how to develop plug-in](docs/Guardium%20Insights/3.2.x/developing_plugins_gi.md).

Please refer [how to manage plug-ins](docs/Guardium%20Insights/3.2.x/Plugins_management.md) to make sure the needed plug-in is installed before configuring a new Universal Connector.

> Once the plug-in is installed you can [create a new connection](docs/Guardium Insights/3.2.x/UC_Configuration_GI.md). 

Reviewer: Link isn't clickable

> The connection initialization can take few minutes, to monitor the current status of connection and the latest traffic click [here](docs/Guardium Insights/3.2.x/monitoring_GI.MD).

Reviewer: Same

# Guardium Insights 3.2.x
The Universal Connector uses plug-in to the datasource server.

> There are [available plugins](../../../docs/available_plugins.md) that are ready for connecting the datasource to Guardium Insights.

Reviewer 2: ```suggestion
There are [available plugins](../../../docs/available_plugins.md) that are ready for connecting the data source to Guardium Insights.
```

> In case that the requested datasource isn't available, please refer [how to develop plug-in](../../../docs/Guardium%20Insights/3.2.x/developing_plugins_gi.md).

Reviewer 2: ```suggestion
In cases where the requested data source isn't available, please refer tp [Developing a plug-in](../../../docs/Guardium%20Insights/3.2.x/developing_plugins_gi.md).
```

Reviewer 2: If it's a new readme you are adding now, please change title to "Developing a plug-in"

> Please refer [how to manage plug-ins](../../../docs/Guardium%20Insights/3.2.x/Plugins_management.md) to make sure the needed plug-in is installed before configuring a new Universal Connector.

Reviewer 2: ```suggestion
Please refer to [Managing plug-ins](../../../docs/Guardium%20Insights/3.2.x/Plugins_management.md) to make sure the required plug-in is installed before configuring a new universal connector.
```

Reviewer 2: If it's a new readme you are adding now, please change title to "Managing plug-ins"

Once the plug-in is installed you can [create a new connection](../../../docs/Guardium%20Insights/3.2.x/UC_Configuration_GI.md). 

> The connection initialization can take few minutes, to monitor the current status of connection and the latest traffic click [here](../../../docs/Guardium%20Insights/3.2.x/monitoring_GI.MD).

Reviewer 2: ```suggestion
The connection initialization can take few minutes. To monitor the current status of connection and the latest traffic, click [here](../../../docs/Guardium%20Insights/3.2.x/monitoring_GI.MD).
```
